lua/barbar/render.lua

```diff
@@ -43,52 +44,52 @@ local ELLIPSIS_LEN = strwidth(ELLIPSIS)
 local last_tabline = ''
 
 --- Concatenates some `groups` into a valid string.
---- @param groups barbar.render.group[]
+--- @param items barbar.render.group.item[]
 --- @return string
-local function groups_to_string(groups)
+local function items_to_string(items)
```

Comment:
I don't like this refactor in this PR, creates a lot of noise.

Reply:
No problem— I'll `drop` the commit and we can revisit it later before finalizing the public-facing API.

Reply:
It's also confusing me because of the group/item name inversion. I'm open to keeping it though, but I don't find the refactor terribly good (but not terribly bad either, the original naming here isn't great).

Reply:
We can probably choose better names anyway. I figured "group of items" was more natural than "item of groups", but perhaps "section of elements" might be better (borrowing from HTML). Not relevant to this specific PR either way